tests/test_functional.py

```diff
@@ -634,8 +634,12 @@ def test_delete_batch(self, user, testapp, unused):
         login(user, testapp)
```

Comment:
Why we don't use var `unused` when we parametrize that var ?

Reply:
We don't want to list specific batches to delete so we use pytest.mark.parametrize only to run the test multiple times and each time we select a random batch to delete.

Right now, it's not possible to parameterize a test based on the content of the database, because the tests are collected before fixtures are executed, see #42

Reply:
aa OK I've got it